card/lib/cardio/commands/rake_command/parser.rb

```diff
@@ -1,14 +1,14 @@
 # -*- encoding : utf-8 -*-
 require "optparse"
 
-module Decko
+module Cardio
   module Commands
     class RakeCommand
       class Parser < OptionParser
         def initialize command, opts
           super() do |parser|
-            parser.banner = "Usage: decko #{command} [options]\n\n" \
-                          "Run decko:#{command} task on the production "\
+            parser.banner = "Usage: card #{command} [options]\n\n" \
+                          "Run card:#{command} task on the production "\
```

Comment:
does this mean monkeys will be seeing help text for the `card` command? We may need to override this in decko.